openapi.yml

```diff
@@ -0,0 +1,506 @@
+        default: stage
+paths:
+  /{repo}/objects/{druid}/lifecycle:
+    get:
+      summary: Retrieve workflow lifecycle for an object
+      responses:
+        '200':
+          description: Workflow lifecycle found
+          content:
+            application/xml:
+              schema:
+                type: string
+      parameters:
+        - name: repo
+          in: path
+          required: true
+          example: dor
+          schema:
+            $ref: '#/components/schemas/Repository'
+        - name: druid
+          in: path
+          required: true
+          example: druid:bc123df4567
+          schema:
+            $ref: '#/components/schemas/Druid'
+        - name: active-only
+          in: query
+          required: false
+          schema:
+            type: boolean
+  /{repo}/objects/{druid}/versionClose:
+    post:
+      summary: Close the current version of the provided druid
+      responses:
+        '200':
+          description: Request succeeded
+      parameters:
+        - name: repo
+          in: path
+          required: true
+          example: dor
+          schema:
+            $ref: '#/components/schemas/Repository'
+        - name: druid
+          in: path
+          required: true
+          example: druid:bc123df4567
+          schema:
+            $ref: '#/components/schemas/Druid'
+        - name: create-accession
+          in: query
+          required: false
+          schema:
+            type: boolean
+        - name: version
+          in: query
+          required: false
+          schema:
+            type: integer 
+  /{repo}/objects/{druid}/workflows:
+    get:
+      summary: Retrieve workflow datastream for an object
+      responses:
+        '200':
+          description: Workflow datastream found
+          content:
+            application/xml:
+              schema:
+                type: string
+      parameters:
+        - name: repo
+          in: path
+          required: true
+          example: dor
+          schema:
+            $ref: '#/components/schemas/Repository'
+        - name: druid
+          in: path
+          required: true
+          example: druid:bc123df4567
+          schema:
+            $ref: '#/components/schemas/Druid'
+    delete:
+      summary: Delete all of the workflows for an object
+      responses:
+        '204':
+          description: No Content
+      parameters:
+        - name: repo
+          in: path
+          required: true
+          example: dor
+          schema:
+            $ref: '#/components/schemas/Repository'
+        - name: druid
+          in: path
+          required: true
+          example: druid:bc123df4567
+          schema:
+            $ref: '#/components/schemas/Druid'
+  /{repo}/objects/{druid}/workflows/{workflow}:
+    get:
+      summary: Show the named workflow for a given druid
+      responses:
+        '200':
+          description: List of workflow steps
+          content:
+            application/xml:
```

Comment:
Same as above. Will stop typing this comment on every XML bit. Though, yeah, it may not be worth it since this is getting into response validation which this PR does not add.